src/server/replica.cc

```diff
@@ -590,7 +441,7 @@ error_code Replica::InitiateDflySync() {
     // Unblock all sockets.
     DefaultErrorHandler(ge);
     for (auto& flow : shard_flows_)
-      flow->CloseSocket();
+      flow->Cancel();
```

Comment:
I'm sure this is the right thing to do, but can you explain why you switched from `CloseSocket()` to `Cancel()` here?

Reply:
I need the call to waker_.Notify() so that the ACK fibers will exit.